### DISCUSSION TITLE
Is there any way to get fzf to split on whitespace rather than newline?

### DISCUSSION THREAD

**Discussion by sss-ng** [junegunn/fzf#4024] [Q&A]
I really enjoy `fzf`, I use it all day every day :) Thanks for the awesome tool.

I was wondering if there is a cleaner way to achieve the following without using `tr ' ' '\n'`?
```bash
action=$(echo "plan" "seed" "up" "init" | tr ' ' '\n' | fzf)
```

**Comment by LangLangBart** [comment]
No, as of 2024 this has to be done with a separate filter.

**Comment by junegunn** [comment]
This is more of a general shell scripting question rather than an fzf question. You want to make echo print each argument on a separate line. You can use `printf` instead.

```sh
printf '%s\n' "plan" "seed" "up" "init" | fzf
```
